drivers/led_strip/ws2812_gpio.c

```diff
@@ -68,24 +50,26 @@ struct ws2812_gpio_cfg {
 #define SET_HIGH "str %[p], [%[r], #0]\n" /* OUTSET = BIT(LED_PIN) */
 #define SET_LOW "str %[p], [%[r], #4]\n"  /* OUTCLR = BIT(LED_PIN) */
 
+#define NOPS(i, unused) "nop\n"
+
 /* Send out a 1 bit's pulse */
-#define ONE_BIT(base, pin) do {			\
-	__asm volatile (SET_HIGH			\
-			DELAY_T1H			\
-			SET_LOW			\
-			DELAY_TxL			\
-			::				\
-			[r] "l" (base),		\
+#define ONE_BIT(base, pin) do {					\
+	__asm volatile (SET_HIGH				\
+			LISTIFY(CONFIG_DELAY_T1H, NOPS, ())	\
```

Comment:
Not sure what to do with these check warnings.

Reply:
It seems to be a false-positive detection.
We can ignore it, but I tried using this patch and solved it.
If this modification is correct, please consider applying.
You can run the check locally with
`./scripts/ci/check_compliance.py --annotate -e KconfigBasic -c HEAD~1..` command.

```
diff --git a/drivers/led_strip/ws2812_gpio.c b/drivers/led_strip/ws2812_gpio.c
index b98f0ba5d22..2024122e2c5 100644
--- a/drivers/led_strip/ws2812_gpio.c
+++ b/drivers/led_strip/ws2812_gpio.c
@@ -51,23 +51,20 @@ struct ws2812_gpio_cfg {
 #define SET_LOW "str %[p], [%[r], #4]\n"  /* OUTCLR = BIT(LED_PIN) */

 #define NOPS(i, unused) "nop\n"
+#define NOP_N_TIMES(n) LISTIFY(n, NOPS, ())

 /* Send out a 1 bit's pulse */
 #define ONE_BIT(base, pin) do {                                        \
-       __asm volatile (SET_HIGH                                \
-                       LISTIFY(CONFIG_DELAY_T1H, NOPS, ())     \
-                       SET_LOW                                 \
-                       LISTIFY(CONFIG_DELAY_T1L, NOPS, ())     \
+       __asm volatile (SET_HIGH NOP_N_TIMES(CONFIG_DELAY_T1H)  \
+                       SET_LOW NOP_N_TIMES(CONFIG_DELAY_T1L)   \
                        ::                                      \
                        [r] "l" (base),                         \
                        [p] "l" (pin)); } while (false)

 /* Send out a 0 bit's pulse */
 #define ZERO_BIT(base, pin) do {                               \
-       __asm volatile (SET_HIGH                                \
-                       LISTIFY(CONFIG_DELAY_T0H, NOPS, ())     \
-                       SET_LOW                                 \
-                       LISTIFY(CONFIG_DELAY_T0L, NOPS, ())     \
+       __asm volatile (SET_HIGH NOP_N_TIMES(CONFIG_DELAY_T0H)  \
+                       SET_LOW NOP_N_TIMES(CONFIG_DELAY_T0L)   \
                        ::                                      \
                        [r] "l" (base),                         \
                        [p] "l" (pin)); } while (false)
```